### PR TITLE
Add testing section to the readme

### PR DESCRIPTION
**Pull request by SamVerschueren**
Improvements are always welcome.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -11,6 +11,7 @@
 - Config and cache handling built-in.
 - Fetching remote files with optional caching.
 - Publish your workflow to npm.
+- Easy [testing](https://github.com/SamVerschueren/alfy-test).
```
Comment (SamVerschueren) [inline readme.md]: Should we point to the `## Testing` section instead?

Comment (sindresorhus) [inline readme.md]: Yes

File: readme.md [hunk 2]
```diff
@@ -128,6 +129,29 @@ $ npm install --global alfred-unicorn
 > Tip: instead of manually updating every workflow yourself, use the [alfred-updater](https://github.com/SamVerschueren/alfred-updater) workflow to do that for you.
 
 
+## Testing
+
+Workflows can be easily tested with [alfy-test](https://github.com/SamVerschueren/alfy-test). Here is a small example.
```
Comment (SamVerschueren) [inline readme.md]: `can be easily tested` or `can easily be tested`?

Comment (sindresorhus) [inline readme.md]: Latter

File: readme.md [hunk 3]
```diff
@@ -12,6 +12,7 @@
 - Fetching remote files with optional caching.
 - Publish your workflow to npm.
 - Automatic [update notifications](#update-notifications).
+- Easy [testing](#testing).
```
Comment (SamVerschueren) [inline readme.md]: Is this ok? Could be

- Easy to [test workflows](#testing).
- Easily [testable workflows](#testing).

Comment (sindresorhus) [inline readme.md]: Last one

**Comment by SamVerschueren** [comment]
Think this is finally good to merge :).

**Comment by sindresorhus** [comment]
Nice work!

**Pull request merged**